### PR TITLE
feat: adding custom label handling for inputs

### PR DESCRIPTION
**Pull request by DeirdreCleary**
# Why
The user should be able to set the labels they wish to use in their iframe, in particular as it will allow localization

# How
Passing all config fields from Inputs initialization to iFrame

### CODE REVIEW

**Review by jeremywagemans — CHANGES_REQUESTED** [review]
Just added a small comment there!

File: lib/v2/core/inputs.js [hunk 1]
```diff
@@ -41,6 +39,14 @@ module.exports = (config) => {
             );
           }
         },
+        setLabels: (labels) =>
```
[the remark this answers is not in the document]
Comment (DeirdreCleary) [reply lib/v2/core/inputs.js]: Good catch! Updated

**Review by jeremywagemans — APPROVED** [review]
Great work on this one!

**Pull request merged**